target/linux/ramips/image/mt7620.mk

```diff
@@ -974,6 +974,25 @@ define Device/tplink_re200-v1
 endef
 TARGET_DEVICES += tplink_re200-v1
 
+define Device/tplink_re210-v1
+  SOC := mt7620a
+  DEVICE_VENDOR := TP-Link
+  DEVICE_MODEL := RE210
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-mt76x0e
+  IMAGES += factory.bin
+  IMAGE/factory.bin := mktplinkfw factory -e -O
+  IMAGE/sysupgrade.bin := mktplinkfw sysupgrade -e -O | append-metadata
+  IMAGE_SIZE := 7936k
+  KERNEL := $(KERNEL_DTB)
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | tplink-v1-header -e -O
+  TPLINK_HWID := 0x02100001
+  TPLINK_HWREV := 0x1
+  TPLINK_HEADER_VERSION := 1
+  TPLINK_FLASHLAYOUT := 8Mmtk
+endef
+TARGET_DEVICES += tplink_re210-v1
```

Comment:
Hi, be aware of https://patchwork.ozlabs.org/patch/1231065/ which would reduce the entries in this block:

```
define Device/tplink_re210-v1
  $(Device/tplink-v1)
  SOC := mt7620a
  DEVICE_MODEL := RE210
  DEVICE_VARIANT := v1
  DEVICE_PACKAGES := kmod-mt76x0e
  IMAGE_SIZE := 7936k
  TPLINK_HWID := 0x02100001
  TPLINK_FLASHLAYOUT := 8Mmtk
endef
TARGET_DEVICES += tplink_re210-v1
```

Reply:
Has been merged in the meantime.

Reply:
Seen, thx. Will continue to work on this PR in about 2 weeks.